File: Learn the Basic/Pattern_printing/Number_Crown_Pattern.py
def Number_Crown_Pattern(n):
    for i in range(1,n+1):
        # number
        for j in range(1,i+1):
            print(j,end="")
        
        #spaces 
        for j in range(2*(n-i)):
            print(" ",end="")
        
        # number 
        for j in range(i,0,-1):
            print(j,end="")
        print()
n=int(input("Enter the value of n: "))
Number_Crown_Pattern(n)

File: Learn the Basic/Pattern_printing/Number_Crown_Pattern.cpp
#include<iostream>
using namespace std;
void Number_Crown_Pattern(int n){
    int spaces = 2 * (n - 1); 
    for(int i=1;i<=n;i++){
        for(int j=1;j<=i;j++){
            cout<<j;
        }
        for(int j=1;j<=2*(n-i);j++){
            cout<<" ";
        }
        for(int j=i;j>=1;j--){
            cout<<j;
        }
        cout<<endl;
        spaces=spaces-2;
    }
}
int main(){
    int n;
    cout<<"Enter the value of n: ";
    cin>> n;
    Number_Crown_Pattern(n);
    return 0;
}